bluesky/callbacks/zmq.py

```diff
@@ -146,7 +146,7 @@ def __init__(self, in_port=None, out_port=None, *, zmq=None):
                 out_port = backend.bind_to_random_port("tcp://*")
             else:
                 backend.bind("tcp://*:%d" % out_port)
-        except:
+        except BaseException:
```

Comment:
Is the intention here to catch all exception from that [hierarchy](https://docs.python.org/3/library/exceptions.html#exception-hierarchy), including `SystemExit`, `KeyboardInterrupt` and `GeneratorExit`?

Reply:
I guess this was the idea. At least this is how the code used to work until now.

Reply:
Yes.

Reply:
OK, thanks. Resolving this point.